src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -178,6 +179,29 @@ public Collection<MergeReviewConceptVersions> getMergeReviewConflictingConcepts(
 		return conflicts.values();
 	}
 
+	private boolean isTargetConceptVersionBehind(Concept sourceVersion, Concept targetVersion) {
+		Set<Integer> sourceEffectiveTimes = new HashSet<>();
+		sourceEffectiveTimes.add(sourceVersion.getReleasedEffectiveTime());
+		sourceEffectiveTimes.addAll(sourceVersion.getRelationships().stream().map(Relationship::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getDescriptions().stream().map(Description::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getDescriptions().stream().map(Description::getAssociationTargetMembers).flatMap(List::stream).map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getAllOwlAxiomMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getInactivationIndicatorMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getAssociationTargetMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		while (sourceEffectiveTimes.remove(null)) ;
```

Comment:
Not sure why do we need while here.

Reply:
This was to remove null values. I have tweaked the code to only add versions if the component is released.